@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1456,8 +1459,11 @@ export class TileGeometryCreator {
                 geoBox: tile.geoBox
             };
             object.userData = tileDisplacementMap;
-        } else if (isSolidLineTechnique(technique)) {
-            object.userData = srcGeometry.objInfos!;
+            // ### PoC this breaks the line picking
```

Comment:
I don't think it breaks line picking.
Line picking as implemented in `RoadPicker` is based purely on `decodedTile.tileInfo`.

On the other hand, i don't know origins of this specific `userData` overwrite commented here.